assignment/index.html

```diff
@@ -58,17 +58,53 @@
        your best to use them to organize your code.
       ===================== */
 
-
       /* =====================
 
       Start code
 
       ===================== */
 
+
+      var healthCentersList = Array(healthCenters.length+1);
+      for(i = 0; i<healthCenters.length; i++){
+        var str = Object.entries(healthCenters[i]).toString();
+        var words = str.split(',');
+        var Length = words.length+1;
+        var keys = new Array();
+        var entries = new Array();
+        for(j = 0; j<words.length; j++){
+          if(j%2 == 0){
+            keys.push(words[j]);
+          }else{
+            entries.push(words[j]);
+          }
+        }
+        healthCentersList[0] = keys;
```

Comment:
This totally works but you'll note that pushing the keys into your array `keys` only needs to happen once. Every time this loop runs, it will overwrite the 0th element in `healthCentersList`.